media/js/firefox/new/compare-edge-experiment.js

```diff
@@ -46,9 +46,8 @@
         var cop = new Mozilla.TrafficCop({
             id: 'experiment_firefox_new_edge',
```

Comment:
We should update the experiment ID for saftey's sake. I doubt we'll see many users returning through a paid media buy like this but if they do we don't want traffic cop sending them back to `v=1` because that's in their stored cookie.

Reply:
nice call 👍